composer.json

```diff
@@ -23,7 +23,9 @@
   "require": {
     "coenjacobs/mozart": "^0.7.1",
     "id4me/id4me-rp": "^1.2",
-    "firebase/php-jwt": "^5.2"
+    "firebase/php-jwt": "^5.2",
+    "symfony/event-dispatcher": "^4",
+    "symfony/console": "^4"
```

Comment:
is this for static analysis?

Reply:
No it is being pulled in by `coenjacobs/mozart`, but I guess that could actually be a dev dependency only.

Reply:
Hm ok, but then we cannot built the autoloader anymore with out the dev depencenies :/

Reply:
Just found https://github.com/nextcloud/mail/pull/4633 let me prepare something similar for that here as well

Reply:
Made an independent PR at https://github.com/nextcloud/user_oidc/pull/296